roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -113,6 +113,40 @@
     age: "-{{ 60 * 24 * 7 * 2 }}"  # two weeks
     exclude: "{{ linux_allowed_large_files + linux_additional_allowed_large_files }}"
 
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Disk: logrotate: Are there files in /var/log that are not rotated?"
+  ansible.builtin.shell: "cat /var/lib/logrotate/logrotate.status"
+  register: logrotate_output
+  tags:
+    - skip_ansible_lint
```

Comment:
Please don't disable ansible-lint for an task entirely. Instead, please provide a `# noqa` comment naming the exact violation that is ignored and provide a reasoning for it.

Though I guess there should not be any violations if slurp is used.